TheMovieDatabase/ViewControllers/Favorites/View/FavoritesViewController.swift

```diff
@@ -58,6 +59,23 @@ final class FavoritesViewController: UIViewController {
         tableView.register(UINib(nibName: MovieTableViewCell.identifier, bundle: nil),
                            forCellReuseIdentifier: MovieTableViewCell.identifier)
         tableView.tableFooterView = UIView()
+
+        refreshControl.attributedTitle = NSAttributedString(string: "Pull to refresh")
+        refreshControl.addTarget(self, action: #selector(refreshData), for: .valueChanged)
+        tableView.refreshControl = refreshControl
+    }
+
+    @objc
+    private func refreshData() {
+        let service = MoviesService()
+        service.loadFavoriteMovies(accountId: accountId) { (movies) in
+            guard let movies = movies else {
+                return
+            }
+            self.movies = movies
+            self.tableView.reloadData()
+            self.refreshControl.endRefreshing()
+        }
```

Comment:
Looks like a duplicate of `loadFavoriteMovies`
What do you think about unification?

Reply:
nice ideaa